### DISCUSSION TITLE
What is an inactive_sections?

### DISCUSSION THREAD

**Discussion by SephVelut** [nvim-lualine/lualine.nvim#667] [Q&A] [answered]
Not documented?

**Comment by fitrh** [comment, marked as answer]
Section to put your component which is displayed when the window is not focused

**Reply by fitrh** [reply]
![20220427_000330_full](https://user-images.githubusercontent.com/55179750/165343612-c31a3a19-3013-4c51-b7c3-b659df324d08.jpg)

**Reply by nyngwang** [reply]
Is it possible to disable lualine for all inactive windows?

**Reply by fitrh** [reply]
Since `'laststatus'` is a global option, I think the best we can do to imitate
a **disabled statusline** is to set inactive highlight to `Normal` and
`inactive_sections.lualine_*` to empty

**Reply by nyngwang** [reply]
I just realized that there is an option `vim.opt.laststatus = 3` that is exactly what I want. Still appreciate your attempt to help :)

**Reply by fitrh** [reply]
Ah, so you mean single status line 

Yes, that option is available for a while now, from lualine, you can use `options.globalstatus = true`, which is the same as `vim.opt.laststatus = 3`
